translator/internaldata/metrics_to_oc.go

```diff
@@ -50,7 +50,7 @@ func MetricDataToOC(md data.MetricData) []consumerdata.MetricsData {
 		ocMetricsData := consumerdata.MetricsData{}
 		ocMetricsData.Node, ocMetricsData.Resource = internalResourceToOC(rm.Resource())
 		ilms := rm.InstrumentationLibraryMetrics()
-		if ilms.Len() == 0 {
+		if ilms.Len() == 0 || ilms.Get(0).Metrics().Len() == 0 {
```

Comment:
What if `ilms.Len()>1`? I think this will work incorrectly.

Reply:
Nice catch, thanks. Fixed.